src/mainwin.cc

```diff
@@ -567,6 +576,140 @@ MainWindow::MainWindow(const QStringList &filenames)
 	viewModeThrownTogether();
 	show();
 
+
+	//Define every shortcut
+
+	std::string absolutePath = PlatformUtils::resourceBasePath()+"/shortcuts/shortcuts.json";
+	QString finalPath = QString::fromLocal8Bit(absolutePath.c_str());
+
+	//reading json file to get shortcuts
+	QFile jsonFile(finalPath);
+	if (!jsonFile.open(QIODevice::ReadOnly | QIODevice::Text))
+	{
+	qDebug() << "Unable to open file, exiting...";
```

Comment:
I think we did not use `qDebug()` so far. There's a separate debug logic that can be enabled via `--debug=<c++-filename>`. See `PRINTD("text");` and `PRINTDB("string %s, %s", param1 % param2);` macros.

Reply:
I removed this, as it was only needed for debugging.